Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -10,15 +10,23 @@
 
 #include "Common/CommonPaths.h"
 #include "Common/FileUtil.h"
-#include "Common/IniFile.h"
 #include "Common/Logging/ConsoleListener.h"
 #include "Common/Logging/Log.h"
 #include "Common/Logging/LogManager.h"
 #include "Common/StringUtil.h"
 #include "Common/Timer.h"
+#include "Core/Config/Config.h"
 
 constexpr size_t MAX_MSGLEN = 1024;
 
+const Config::ConfigInfo<bool> LOGGER_WRITE_TO_FILE{
```

Comment:
Yeah, that's the big question I have with this. The difference with log settings is that 1) they're all confined to this file anyways and 2) I didn't want to make settings for each log type. Is making config infos globally accessible (e.g. a "LoggerSettings.h") an architectural decision, or was it just a good fit for graphics settings?